zuul.d/nodeset.yaml

```diff
@@ -87,6 +87,34 @@
       - name: ocps
         nodes:
           - crc
+      - name: rh-subscription
+        nodes:
+          - standalone
+
+- nodeset:
+    name: centos-9-multinode-rhel-9-2-crc-extracted-2.30-3xl
```

Comment:
@lewisdenny can you help me spot if i missed something here?

zuul is not happy with that. I got 2 node failures in a row just now (no logs)

```
data-plane-adoption-osp17-multinode-to-extracted-crc https://review.rdoproject.org/zuul/build/0f3be7a0efdc4135969e22554bf8013c : NODE_FAILURE Node request 200-0006921783 failed in 0s

data-plane-adoption-osp17-multinode-to-extracted-crc https://review.rdoproject.org/zuul/build/fe43d52d654849b78b983f088cf732f7 : NODE_FAILURE Node request 200-0006921980 failed in 0s
```

https://review.rdoproject.org/r/c/testproject/+/51274/2#message-bde3a529c831efc1b99862ce2ee297b9ce01aed1

Perhaps we need this nodeset to merge first. I am going to restore to how it was for now and see if that sorts the error out or if there is some ongoing wider issue

hmmm... actually it might be some wider issue https://review.rdoproject.org/zuul/builds?result=NODE_FAILURE&skip=0

i see quite a few from today ...  :/

Reply:
@marios it's happening to all adoption jobs, I pinged you in a thread discussing it in slack, let me add you to another one

Reply:
thanks Joan, looks like that issue should be clear now.

I am going to restore the nodeset definition to nodesets.yaml then

 pushing update momentarily